cmd/controller/main.go

```diff
@@ -86,10 +88,21 @@ func main() {
 	pvcInformer := k8sInformerFactory.Core().V1().PersistentVolumeClaims()
 	podInformer := k8sInformerFactory.Core().V1().Pods()
 
+	imageFactory := &registry.ImageFactory{
+		KeychainFactory: secret.NewSecretKeychainFactory(k8sClient),
+	}
+
+	imageUtilFactory := &cnb.ImageFactory{
+		KeychainFactory: secret.NewSecretKeychainFactory(k8sClient),
+	}
+
 	metadataRetriever := &cnb.RemoteMetadataRetriever{
-		RemoteImageFactory: &registry.ImageFactory{
-			KeychainFactory: secret.NewSecretKeychainFactory(k8sClient),
-		},
+		RemoteImageFactory: imageFactory,
+	}
+
+	rebaser := cnb.ImageRebaser{
+		RemoteImageFactory: imageUtilFactory,
+		Rebaser:            lifecycle.Rebaser{Logger: logging.New(os.Stderr)},
```

Comment:
Not sure what to do here. The lifecycle expects a lifecycle logger and not a sugared logger. This will write to the controller logs. I am not certain what we want. We should talk about what we wanna do in this situation.

Reply:
+1 to wrapping our zap logger.